ShopApp/Data/API/Alamofire/Routers/BaseRouter.swift

```diff
@@ -9,6 +9,8 @@
 import Alamofire
 import ShopApp_Gateway
 
+let kCacheControlMaxAgeKey = "Cache-Control-Max-Age"
```

Comment:
Move constant to class

Reply:
Base API class uses this constant.

Reply:
That's fine. You can make it `static` and get access as `BaseRouter.kCacheControlMaxAgeKey `